examples/tutorial/c_api_tutorial.c

```diff
@@ -12,6 +12,286 @@
 // Modify this path if necessary.
 const char *pmem_path = "/mnt/pmem0/tutorial_kvdk_example";
 
+static int CmpCompare(const char *a, size_t alen, const char *b, size_t blen) {
```

Comment:
StrCmp or StringCompare

Reply:
not limited string. Reference leveldb: https://github.com/google/leveldb/blob/master/db/c_test.c#L107